src/bin/common/parse_command.ml

```diff
@@ -496,8 +523,9 @@ let halt_opt version_info where =
     | Some w -> handle_where w; `Ok true
     | None -> if version_info then (handle_version_info version_info; `Ok true)
       else `Ok false
-  with Failure f -> `Error (false, f)
-     | Error (b, m) -> `Error (b, m)
+  with
+  | Failure f -> `Error (false, pretty_error f) (* S: dead code? *)
```

Comment:
Nit: Having comments with questions/remarks on the code is good, but we should avoid using obscure shorthands in such notes --- using a full name or an explicit handle is better (I personally use `TODO(bclement):`).